chrome/content/zotero/preferences/preferences_export.xul

```diff
@@ -49,10 +50,15 @@
 			<label value="&zotero.preferences.quickCopy.defaultOutputFormat;" control="quickCopy-menu"/>
 			<menulist id="zotero-quickCopy-menu"/>
 			
-			<separator class="thin"/>
-			
-			<checkbox id="zotero-quickCopy-copyAsHTML" label="&zotero.preferences.quickCopy.copyAsHTML;"
-				oncommand="Zotero_Preferences.Export.buildQuickCopyFormatDropDown(document.getElementById('zotero-quickCopy-menu'), this.checked ? 'html' : '');"/>
+			<hbox align="center">
+				<label value="&zotero.preferences.quickCopy.locale;" control="locale-menu"/>
+				<menulist id="locale-menu"/>
```

Comment:
Doesn't look like that deals with multiple "locale-menu" menulists either. Basically, what I'm asking is whether any code, other than preferences_export.js (which you would need to adjust the ID values in), would break if this ID would be changed. This would (hypothetically) be the case if you passed the `document` object to `populateLocaleList` and were trying to get the menu list from within `populateLocaleList` by doing something like `doc.getElementById('locale-menu')`. Then, all of your locale menus would have to have the same ID.